ruby/ql/lib/codeql/ruby/controlflow/CfgNodes.qll

```diff
@@ -615,6 +615,22 @@ module ExprNodes {
     final override VariableReadAccess getExpr() { result = ExprCfgNode.super.getExpr() }
   }
 
+  private class InstanceVariableAccessMapping extends ExprChildMapping, InstanceVariableAccess {
+    override predicate relevantChild(AstNode n) { n = this.getSelfVariableAccess() }
+  }
+
+  /** A control-flow node that wraps a `InstanceVariableAccess` AST expression. */
```

Comment:
an

Reply:
Done, I also spotted another case like this for `InstanceVariableWriteAccessCfgNode` which made me realise I should probably use that class in the `storeStep` implementation.